### ISSUE TITLE
Feature request "Taildrop" 

### ISSUE THREAD

**Issue by bolavefasfas** [netbirdio/netbird#528]
Can you add this feature in netbird like birddrop 
similar to this 
**Taildrop**
https://tailscale.com/kb/1106/taildrop/
and 
**Exit node** 
https://tailscale.com/kb/1103/exit-nodes/

**Comment by dgkovachev** [comment]
if I get to chose, I would vote for the exit node to be made available first!

**Comment by bolavefasfas** [comment]
Any update on this feature if it will be implemented or any eta ?

**Comment by bolavefasfas** [comment]
@surik I cant see any similar feature for taildrop in the [public roadmap ](https://github.com/netbirdio/netbird/projects/2 ) of netbird.

Please do add it its fine even its later at the end of this year please.

**Comment by europacafe** [comment]
+1. I use it a lot to transfer files between my desktop and mobile phones. If this feature is implemented, I could stop using Tailscale!

**Comment by bolavefasfas** [comment]
> +1. I use it a lot to transfer files between my desktop and mobile phones. If this feature is implemented, I could stop using Tailscale!

Yes Please Idk how to make this issue heard.

 If this feature is implemented, I could stop using Tailscale! - Me as well.

**Comment by sahara101** [comment]
+1 from me

**Comment by Marcus1Pierce** [comment]
@bolavefasfas Exit Node is now available in netbird. https://docs.netbird.io/how-to/configuring-default-routes-for-internet-traffic.

And maybe for Taildrop alternative, you can try [PairDrop](https://pairdrop.net/) for now.

**Comment by bolavefasfas** [comment]
Thank you for implementing the exit node feature. Amazing Feat !!

My primary use case involves sending files to two locations. I'd like a feature similar to Taildrop, which isn't exposed to the internet. However, for something like Pairdrop, I would need to host it on a server and connect it via Netbird, which adds complexity and isn't as convenient. Moreover, requiring users to open a browser for file sharing isn't ideal.

Could you consider adding a file-sharing feature similar to Taildrop in Netbird? It would be extremely useful and a great tool for users. Additionally, it would be an improvement if notifications were included for events like 'file sent' or 'file received,' as Taildrop currently lacks this. If these features are added, it would make Netbird even better than Tailscale."

**Comment by Marcus1Pierce** [comment]
At that point, i need that feature too. And looks like we will have to wait for the next update, hoping that the feature will be implemented soon.

By the way, I'm not the developer.

**Comment by bolavefasfas** [comment]
> At that point, i need that feature too. And looks like we will have to wait for the next update, hoping that the feature will be implemented soon.
> 
> By the way, I'm not the developer.

I agree like it will just break the apple samsung and all c;osed source ecosystem.
just imagine you can send files from any device to any device like nice and no need to store it on cloud like google drive (painful)

**Comment by bolavefasfas** [comment]
Still Waiting on Feature similar to Taildrop, its been more than 2 years now.

**Comment by bolavefasfas** [comment]
> [@bolavefasfas](https://github.com/bolavefasfas) Exit Node is now available in netbird. https://docs.netbird.io/how-to/configuring-default-routes-for-internet-traffic.
> 
> And maybe for Taildrop alternative, you can try [PairDrop](https://pairdrop.net/) for now.
it isn't as intuitive as taildrop 


Right click and sent to whatever device list you want to send it to its almost perfect - con does not notify the other device/user that i file has been received. 
<img width="324" height="268" alt="Image" src="https://github.com/user-attachments/assets/b9d6bf68-ba9c-4fc3-adc8-1373fb8ec333" />

**Comment by nazarewk** [comment]
We are discussing the implementation of such feature at https://github.com/netbirdio/netbird/issues/4204 , let's move discussion there.
